bcipy/gui/gui_main.py

```diff
@@ -1,106 +1,215 @@
 import os
-import wx
-import wx.lib.agw.gradientbutton as GB
-import wx.lib.agw.aquabutton as AB
-import wx.lib.buttons as buttons
+import sys
+import logging
 
+from enum import Enum
 
-class BCIGui(wx.Frame):
-    """BCIGui."""
+from PyQt5.QtWidgets import (
+    QMainWindow,
+    QApplication,
+    QFileDialog,
+    QWidget,
+    QPushButton,
+    QLabel,
+    QLineEdit,
+    QComboBox,
+    QGridLayout,
+    QMessageBox)
+from PyQt5.QtGui import QPixmap, QFont
+from PyQt5.QtCore import pyqtSlot
```

Comment:
Are we pulling in pyqt transitively? We may want to add it to our requirements.txt file.